t/57-oracle-datetime.t

```diff
@@ -83,6 +89,73 @@ isa-ok $sth.column-types[0], DateTime, 'CURRENT_TIMESTAMP is DateTime';
 $sth.execute;
 my $datetime2 = $sth.row[0];
 isnt $datetime, $datetime2,		    'Server drift';
+
 diag $datetime2.Instant - $datetime.Instant;
 
-$dbh.execute($dropper);
+ok $dbh.execute('DELETE FROM test_datetime WHERE adate IS NOT NULL'), 'Delete rows';
+ok $dbh.dispose, 'Close session';
+
+note '#+ -------------------------------------------------------- +';
+note '## Issue #204 - ADDED Missing TIMESTAMP Support (BEGIN)';
```

Comment:
I prefer 'add' to 'added' according to the git-book regarding commit messages.

Reply:
feel free to change to comment all you like; Seem hardly worth the effort.

Reply:
Frankly I'd completely lose the note in the file, it serves no useful purpose - most people won't see the message and anyone who is interested can see the commit message,

Reply:
notes like this serve  two purposes
- It acts like a script comment
- It demarcates sections of a test; the next poor sap making changes will have yet another clue about what is taking place

I was tempted to write it as a subtest which would do something similar; but I didn't see it being used